JASP-Engine/JASP/R/auditClassicalBenfordsLaw.R

```diff
@@ -582,18 +582,18 @@ auditClassicalBenfordsLaw <- function(jaspResults, dataset, options, ...){
   approve <- state[["pvalue"]] >= (1 - options[["confidence"]])
 
   conclusion <- ifelse(approve, no = gettext("can be rejected"), yes = gettext("can not be rejected"))
-  confidenceLabel <- paste0(round(options[["confidence"]] * 100, 2), "%")
   
   pvalue <- round(state[["pvalue"]], 4)
   if(pvalue < 0.01)
     pvalue <- "< .01"
+  pvalue <- ifelse(approve, no = paste0(pvalue, " < \u03B1"), yes = paste0(pvalue, " >= \u03B1"))
 
   distribution <- base::switch(options[["distribution"]], "benford" = "Benford's law", "uniform" = "the uniform distribution")
 
   conclusionText <- base::switch(options[["digits"]],
-                                  "first" = gettextf("The <i>p</i> value is determined to be %1$s. Therefore, the null hypothesis that the distribution of first numbers in the population conforms to %2$s <b>%3$s</b> with <b>%4$s</b> confidence.", pvalue, distribution, conclusion, confidenceLabel),
-                                  "firstSecond" = gettextf("The <i>p</i> value is determined to be %1$s. Therefore, the null hypothesis that the distribution of the two first numbers in the population conforms to %2$s <b>%3$s</b> with <b>%4$s</b> confidence.", pvalue, distribution, conclusion, confidenceLabel),
-                                  "last" = gettextf("The <i>p</i> value is determined to be %1$s. Therefore, the null hypothesis that the distribution of last numbers in the population conforms to %2$s <b>%3$s</b> with <b>%4$s</b> confidence.", pvalue, distribution, conclusion, confidenceLabel))
+                                  "first" = gettextf("The <i>p</i> value is observed to be %1$s and the null hypothesis that the distribution of first digits in the data set conforms to %2$s <b>%3$s</b>.", pvalue, distribution, conclusion),
```

Comment:
"The <i>p</i> value is %1$s and the null hypothesis that the first digits in the data set is distributed according to %2$s <b>%3$s</b>."

Ex. "The <i>p</i> value is 0.04 and the null hypothesis that the first digits in the data set is distributed according to Benford's law is rejected."

Ex. "The <i>p</i> value is 0.96 and the null hypothesis that the first digits in the data set is distributed according to Benford's law is not rejected."